fsw/cfe-core/src/es/cfe_es_api.c

```diff
@@ -1633,6 +1638,11 @@ uint32 CFE_ES_CalculateCRC(const void *DataPtr, size_t DataLength, uint32 InputC
     const uint8 *BufPtr;
     uint8  ByteValue;
 
+    if (DataPtr == NULL || DataLength == 0)
```

Comment:
Suggest removing DataLength == 0 check, won't crash... will just return the input CRC (which isn't an error).

Reply:
All this check is already doing is returning the input crc, it is not reporting an error.